topbeat/docs/topbeat-filtering.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[filtering-data]]
+=== Filtering the Exported Data
+
+When your use case requires only a subset of the data exported by Topbeat, you can <<topbeat-filtering-overview,use Topbeat config options to filter the data>>, or you can <<generic-filtering-overview-topbeat,use the generic filtering provided by libbeat>>.
+
+[[topbeat-filtering-overview]]
+==== Topbeat Config Options for Filtering
+
+// Not sure if this is technically speaking a filtering option, but it seemed like a candidate since it uses regexp.
```

Comment:
I think it's good to include it in this section. :+1:  I think the comment can be removed.